start_server.py

```
@@ -0,0 +1,93 @@
+#!/usr/bin/env python3
+"""
+Start the ch11.ai API server with controlled output
+"""
+
+import subprocess
+import time
```

Ruff (0.11.9)

`time` imported but unused

Remove unused import: `time`

(F401)